README.md

```diff
@@ -32,6 +32,35 @@ at the Singapore Haskell meetup: http://typeful.net/talks/hpack
 
 ## Examples
 
```

Comment:
It may be worth acknowledging in the introduction that users of Stack have access to templates for projects that include `package.yaml` files, but users of Cabal (the tool) may find the minimal example useful.